demoapp/src/main/java/me/uport/sdk/demoapp/SignJWTListActivity.kt

```diff
@@ -0,0 +1,33 @@
+package me.uport.sdk.demoapp
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.ArrayAdapter
+import android.widget.ListAdapter
+import kotlinx.android.synthetic.main.activity_list_main.*
+
+class SignJWTListActivity: AppCompatActivity() {
+
+    private val features = arrayOf("Use KeyPair Signer", "Use UportHDSigner with Fingerprint")
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_list_main)
```

Comment:
since this layout is getting reused, it's worth renaming it to something more generic like:
`activity_list`